lms/templates/arbisoft_survey.html

```diff
@@ -36,43 +67,102 @@
 %>
 
 <main id="main" aria-label="Content" tabindex="-1">
-    <div class="container dashboard" id="dashboard-main">
-      <div class="my-courses" id="my-courses">
-        <header class="wrapper-header-courses">
-          <h2 class="header-courses">${_("Please fill this form")}</h2>
-        </header>
-        %if profile.errors:
-            %for field in profile:
-                %for error in field.errors:
-                    <div class="alert alert-danger">
-                        <strong>${field.name} - ${error}</strong>
-                    </div>
+    <div class="login-register">
+      <div class="form-type">
+          <div class="form-wrapper">
+              %if profile.errors or courses.errors or technology.errors:
+                <div class="status submission-error" aria-live="polite">
+                    <h4 class="message-title">We couldn't save your details.</h4>
+                    <ul class="message-copy">
+                        %for field in profile:
+                            %for error in field.errors:
+                                <li>${get_label(field.name)}: ${error}</li>
+                            %endfor
+                        %endfor
+                        %for field in courses:
+                            %for error in field.errors:
+                                <li>${get_label(field.name)}: ${error}</li>
+                            %endfor
+                        %endfor
+                        %for field in technology:
+                            %for error in field.errors:
+                                <li>${get_label(field.name)}: ${error}</li>
+                            %endfor
+                        %endfor
+                    </ul>
+                </div>
+              %endif
+            <form method="POST" action="/arbisoft_survey" class="post-form">
+                <div class="section-title lines">
+                    <h2>
+                        <span class="text">Complete Your Registration</span>
+                    </h2>
+                </div>
+                <input type="hidden" name="csrfmiddlewaretoken" value="${ csrf_token }">
+                %for element in profile:
+                    %if element.name == 'other_studied_course':
+                        <div class="form-field">
+                            %for course in courses:
+                                <label>${course.label}</label>
+                                %for field in course:
+                                    <div class="form-field">
+                                        <label for="${field.id_for_label}">
+                                            ${field.tag()}
```

Comment:
what are you doing for other checkbox, 
are you saving others data in [these fields](https://github.com/amir-qayyum-khan/edx-platform/pull/16/files#diff-55b798ee23a7fde8d1103408afcd0f16)? Please explain.